### ISSUE TITLE
Determine data problems on Day Measures feature of home page

### ISSUE THREAD

**Issue by dcarsey** [RENCI/ctmd#299]
- Day Measures (switch to Timeline Metrics… Turnaround times) - ⇐ easy !
- Proposal Submission date to date of first contact (Intro Call is on the initial consultation additional data elements form)  Find out if these are easy
- Proposal Submission date to date of initial consultation KO meeting (date kick off meeting occurs is on the initial consultation additional data elements form)
- Proposal Submission to date initial consultation complete (work complete is on the initial consultation additional data elements form)
- Proposal Submission to grant submission (already on the CTMD home page) (OK)
- Actual_date contingent on there being a target date tgs_date_2 = actual grant submission ^^
- Grant submission date that Project Leads confirm (for comprehensive?  MJ doublechecking - might also be capturing when studies get resources)
- Grant resubmission also to be considered - for now let’s do submission date to keep it simple

Big changes requested to the Day Measures graphic on home page of CTMD -- including what Duke is interested in for metrics!

Labeling request:  Day Measures (switch to Timeline Metrics or maybe Turnaround times)  (@mbwatson )

**More interested in these metrics - removing 3 of the 4 currently on the CTMD**

Do AVERAGE NUMBER OF DAYS as well as MEDIAN NUMBER OF DAYS

- [ ] Proposal Submission date to date of first contact (Intro Call is on the initial consultation additional data elements form)  Find out if these are easy

- [ ] Proposal Submission date to date of initial consultation KO meeting (date kick off meeting occurs is on the initial consultation additional data elements form)

- [ ] Proposal Submission to date initial consultation complete (work complete is on the initial consultation additional data elements form)

- [x] Proposal Submission to grant submission (already on the CTMD home page) (This one is OK to leave on CTMD.)

- [ ] Added from #249 :  What is the average length of time between the funds release date and the actual protocol final date?  [**Shruti has done all of the prep work for this -- added all the fields necessary to answer the query to the mapping/CTMD. **]  Next up! Matt needs to add this on the front end/home screen.

**Mapping assistance:**

- Actual_date contingent on there being a target date tgs_date_2 = actual grant submission ^^
- Grant submission date that Project Leads confirm (for comprehensive?  MJ doublechecking - might also be capturing when studies get resources)
        (Grant resubmission also to be considered - for now let’s do submission date to keep it simple)

1) Make sure that each of the metrics in Day Measures bar graphs are pulling the correct data.
2) Restore Median Number of Days to CTMD once all of the data is correct.
3) In the labeling, place the word * Proposal * before the word Submission to clarify the difference between Proposal Submission and Grant Submission.

Shruti -- you'll likely be working with Hao on making sure the data is right / accurate and finalized.  Then hand this ticket to Matt to make changes to the graphic.  THANK YOU.

**Comment by dcarsey** [comment]
- [ ] Check with Shruti to confirm accurate mapping
- [ ] Let me know if you need help getting access to the particular form mentioned:  initial consultation data elements form
- [ ] labeling requirements:  submission should be proposal submission; header should be timeline metrics instead of Day Measures
- [ ] Create a set of bar graphs that report average number of days between milestones
- [ ] Create a duplicate set that report median number of days between milestones
- [ ] Ok to make the bar graphs smaller to fit the real estate on the screen better
- [ ] Jackie will sanity check the data
- [ ] Copy/paste/link to this ticket these two future graph requests (see below) in new ticket and label with FY2022:
- [ ] Grant submission date that Project Leads confirm (for comprehensive? MJ doublechecking - might also be capturing when studies get resources)
- [ ] Grant resubmission also to be considered - for now let’s do submission date to keep it simple

**Comment by dcarsey** [comment]
FOR NOW -- 

Change label of the graphic from DAY MEASURES to TIMELINE METRICS

If there's room, insert the word proposal in front of the word submission on two of the bar graphs.  Ok to skip this step if it messes up the appearance.

**Comment by luthfib** [comment]
Timeline Metrics has been changed. @dcarsey What should we do with this ticket?

**Comment by luthfib** [comment]
Closing this ticket, creating a new ticket to address the changes that were not addressed

**Comment by luthfib** [comment]
https://github.com/RENCI/ctmd/issues/299
